src/components/WalletsPage/SideBar/SideBarList.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
Just SideBar should be enough

Reply:
@mod This is not named as a `SideBar` because there is an `./SideBar/index.js` file in folder that corresponds to `SideBar` (component) and `SideBarList`. 

```
 SideBar/
 --./index.js
 --./SideBarList.js
 --./styles.js (this one is shared and used by both index.js (SideBar) and SideBarList)
```